auth/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@microsoft/vscode-azext-azureauth",
-    "version": "4.1.0",
+    "version": "4.1.1",
     "lockfileVersion": 3,
     "requires": true,
     "packages": {
```

Comment:
It shows up a couple lines below too

Reply:
#Embarrassing, should've just used `npm i` instead of trying to do it manually. 😳